code/__DEFINES/mobs.dm

```diff
@@ -448,6 +448,8 @@
 #define FLASH_PROTECTION_NONE 0
 #define FLASH_PROTECTION_FLASH 1
 #define FLASH_PROTECTION_WELDER 2
+#define FLASH_PROTECTION_WELDER_PLUS  3
+#define FLASH_PROTECTION_MAXIMUM  4
```

Comment:
```suggestion
#define FLASH_PROTECTION_WELDER_PLUS 3
#define FLASH_PROTECTION_MAXIMUM 4
```

Reply:
also why not make it `INFINITY` or did i misunderstand what melbert was asking

Reply:
I couldn't get infinity to work so I went with this